exercise_051_cluster_singleton_akka_bootstrap_dns/src/main/resources/application.conf

```diff
@@ -0,0 +1,72 @@
+## Recommended settings for robust cluster
+akka.cluster.shutdown-after-unsuccessful-join-seed-nodes = 30s
+akka.coordinated-shutdown.exit-jvm = on
+
+akka {
+
+  io.dns.resolver = async-dns
+
+  log-config-on-start = on
+  discovery.method = akka-dns
+  management.cluster.bootstrap.contact-point-discovery {
+    service-name = "_picluster._tcp.lb.com"
+    protocol = "tcp"
+    port = "8558"
```

Comment:
'port' is not a valid option here (https://github.com/akka/akka-management/blob/master/cluster-bootstrap/src/main/resources/reference.conf#L25)

Reply:
`port-name` does exist and defaults to `${akka.management.http.port}` which defaults to `8558`, though.

Reply:
@raboof Indeed: DNS discovery uses `A` records by default (as I can see using WireShark).
I wanted to use `SRV` records as this is to demonstrate it in action and I serve DNS records using `dnsmasq`. The latter doesn't support having multiple IP addresses for a single record, so I had to resort to `SRV` records.
Thanks for chasing this. I think the docs should be more clear about this.

Reply:
> I think the docs should be more clear about this.

Could you please open a ticket and/or a PR?

Reply:
@raboof I'll open a ticket and see if I can create a PR.